Documentation/Profiling/davbr-blog-archive/Creating an IL-rewriting profiler.md

```diff
@@ -19,12 +19,12 @@ Keep in mind there are many ways to do this. I'll outline one of the more straig
 
 - In your **ICorProfilerCallback2::ModuleLoadFinished** callback, you call **ICorProfilerInfo2::GetModuleMetadata** to get a pointer to a metadata interface on that module. 
 - QI for the metadata interface you want. Search MSDN for "IMetaDataImport", and grope through the table of contents to find topics on the metadata interfaces. 
-- Once you're in metadata-land, you have access to all the types in the module, including their fields and function prototypes. You may need to parse metadata signatures and [this signature parser](samples/sigparser.cpp) may be of use to you. 
+- Once you're in metadata-land, you have access to all the types in the module, including their fields and function prototypes. You may need to parse metadata signatures and [this signature parser](samples/sigparse.cpp) may be of use to you. 
```

Comment:
BTW: This file does not render correctly on github because of broken new lines. Are you interested in sending a PR to fix it?

Reply:
Sure! #23979